src/frontend/preloads/exportNode.ts

```diff
@@ -1,13 +1,15 @@
-
-import type { IpcRenderer } from 'electron'
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+import { IpcRenderer } from 'electron'
 import { debounce } from 'lodash'
```

Comment:
`import debounce from 'lodash/debounce`

Reply:
Does not work with the webpack config here